kafka/kafka.go

```diff
@@ -316,6 +324,38 @@ func (tps TopicPartitions) Swap(i, j int) {
 	tps[i], tps[j] = tps[j], tps[i]
 }
 
+// Node represents a Kafka broker.
+type Node struct {
+	// Node id.
+	ID int
+	// Node host.
+	Host string
+	// Node port.
+	Port int
+}
+
+func (n Node) String() string {
+	return fmt.Sprintf("[%s:%d]/%d", n.Host, n.Port, n.ID)
+}
+
+// GroupTopicPartitions represents a consumer group's TopicPartitions.
+type GroupTopicPartitions struct {
```

Comment:
We could name it ConsumerGroupTopicPartitions to keep naming consistent with method name, as Pranav is doing in the Python PR. The Java class has the same name but it's not used by the client.